scriptmodules/supplementary/bluetooth.sh

```diff
@@ -255,59 +294,51 @@ function register_bluetooth() {
                 # read "Confirm passkey (yes/no): "
                 echo "yes" >&3
                 read -n 26 line
-                skip_connect=1
                 break
                 ;;
             "DisplayPasskey"*|"DisplayPinCode"*)
                 # extract key from end of line
-                # DisplayPasskey (/org/bluez/1284/hci0/dev_01_02_03_04_05_06, 123456)
-                [[ "$line" =~ ,\ (.+)\) ]] && pin=${BASH_REMATCH[1]}
-                dialog --backtitle "$__backtitle" --infobox "Please enter pin $pin on your bluetooth device" 10 60
+                [[ "$line" =~ ,\ (.+)\) ]] && passkey=${BASH_REMATCH[1]}
+                dialog --backtitle "$__backtitle" --infobox "Please enter passkey $passkey (and press ENTER) on your Bluetooth device now." 10 60
                 ;;
             "Creating device failed"*)
                 error="$line"
                 ;;
+            "Done.")
```

Comment:
Explicit `Done.` check shouldn't be necessary anymore, now that `pair-device` script can't return non-zero exit code under non-error circumstances (due to old "not an input device" case).

Reply:
But this is a moot point anyway, if I can get rid of the Python scripts completely.  This loop/parsing will all need to change anyway to work with the `blue-tools` stuff instead.